block-BJaaaw/code/index.js

```diff
@@ -0,0 +1,190 @@
+// if..else
+
+/*
+1. Write a program that asks the user for a number and check the following conditions:
+
+  - `if` the number is even print the message " number is even"
+  - `if` the number is odd print the message "number is odd"
+*/
+let num = prompt("Enter a number");
+if(num%2==0){
+  alert("Number is even");
+}
+else {
+  alert("number is odd");
+}
```

Comment:
You can also convert the value returned from `prompt` into `number`. Here it will work because `%` operator only works only on numbers.